plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/OTelTracingContextPropagator.java

```diff
@@ -34,13 +39,23 @@ public OTelTracingContextPropagator(OpenTelemetry openTelemetry) {
     @Override
     public Span extract(Map<String, String> props) {
```

Comment:
Not related to this change, but would it make sense to return `Optional<Span>` as well here?